docs/commands.md

```diff
@@ -0,0 +1,185 @@
+# Commands List
+
+The following is a list of commands, their arguments, and what they do.
+
+## Administration Commands
+These are commands for the maintenance of LilyBot. They can only be run by server Admins.
+
+### Set Config
+Name: `config set`
+
+Arguments:
+* `guildId` - ID of the guild to set a config for - Guild ID
+* `moderatorRole` - Moderator role that will be pinged for reports - Role
+* `modActionLog` - Channel for embeds created by mod actions to be sent - Channel
+* `messageLogs` - Channel for deleted messages to be logged - Channel
+* `joinChannel` - Channel for members joining or leaving to be logged - Channel
+* `supportChannel` - Channel for the support auto-threading system to be used in - Optional Channel
+* `supportTeamRole` - The role to be added to any threads created in the `supportchannel` - Optional Role
+
+Result: Sets a configuration in the database for `guildId`.
+
+### Clear Config
+Name: `config clear`
+
+Arguments:
+* `guildId` - ID of the guild to clear the config for - Guild ID
+
+Result: Clears the configuration for `guildId`.
+
+### Presence
+Name: `set-status`
+
+Arguments:
+* `presence` - Value for Lily's status - String
+
+Result: Lily's "Now Playing:" status is set to `presence`.
+
+## Moderation Commands
+These commands are for use by moderators.
+They utilize built-in permission checks.
+All moderation commands are logged to the action log.
```

Comment:
```suggestion
All moderation commands are logged to the set action-log.
A Direct Message is sent to the target user containing the sanction they recieved and the provided reason. If it fails to DM them, it will log this in the action-log embed
```